4001-5000/LIT4145Onausp.xml

```diff
@@ -4,7 +4,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">On auspicious and inauspicious days</title>
+            <title xml:id="t1" xml:lang="gez">ሐሳበ፡ ጽንዕት፡ ወጽንጽንት፡</title>
+            <title corresp="#t1" xml:lang="gez">Ḥasāba ṣǝnǝʿt wa-ṣǝnṣǝnt</title>
+            <title corresp="#t1" xml:lang="en">Calculation of auspicious and inauspicious (days)</title>
+            <title corresp="#t1" xml:lang="fr">Calculation sur les (jours) fastes et néfastes</title>
+            <title xml:id="t2" xml:lang="gez">ሐሳበ፡ ተአምር፡ ጽንዕት፡ ወጽንጽንት፡</title>
+            <title corresp="#t2" xml:lang="gez">Ḥasāba taʾammǝr ṣǝnǝʿt wa-ṣǝnṣǝnt</title>
```

Comment:
Sure about translation of ተአምር፡?

Reply:
An alternative could be: "Reckoning the wonder of auspicious and inauspicious (days)" or  "Evaluation of the sign(s)/signal(s) of auspicious and inauspicious (days)". Is that better?

Reply:
"Reckoning" is better.